infrastructure/storage_stack.py

```diff
@@ -23,6 +24,34 @@ def __init__(self, scope: Construct, stack_id: str, deploy_env: str, **kwargs: A
         else:
             resource_removal_policy = RemovalPolicy.DESTROY
 
+        ############################################################################################
+        # ### DEPLOYMENT VERSION ###################################################################
+        ############################################################################################
+
+        aws_ssm.StringParameter(
+            self,
+            "git-branch",
+            parameter_name=f"/{ENV}/git_branch",
+            string_value=f"{GIT_BRANCH}",
```

Comment:
`f"{GIT_BRANCH}"` could be simplified to just `GIT_BRANCH`. Ditto elsewhere in this file.

Reply:
Thank you, fixed now.